ecommerce-ui/src/App.jsx

```diff
@@ -0,0 +1,120 @@
+import "./App.css";
+import React, { useState, useEffect } from "react";
+import { Routes, Route, Link, BrowserRouter } from "react-router-dom";
+import Home from "./Components/Home";
+import Cart from "./Components/Cart/Cart";
+import Property from "./Components/Property/Property";
+import Properties from "./Components/Properties/Properties";
+import Collection from "./Components/Collections/Collection";
+
+import propertyData from "./_data/bnbs.json";
+import locationData from "./_data/locations.json";
+
+export default function App() {
+  let [cartItems, setCartItems] = useState([]);
```

Comment:
Great job! And yes, this would be a good case to use the Context API + `useContext`. Try it out if you have some time!